### ISSUE TITLE
not compatible with stylelint 13.x

### ISSUE THREAD

**Issue by squarecandy** [WordPress-Coding-Standards/stylelint-config-wordpress#263]
Hi - 

I'm getting a warning when I install the most recent stylelint (`"stylelint": "^13.3.3"`) with stylelint-config-wordpress:

```
npm WARN stylelint-config-wordpress@16.0.0 requires a peer of stylelint@^10.1.0 || ^11.0.0 || ^12.0.0 but none is installed. You must install peer dependencies yourself.
```

Any chance this will be updated soon?

Thanks!

**Comment by ocean90** [comment]
This was updated in https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/commit/c2c1112b9631c10d00d61d87a8beba8a715ea167 but no new release yet.

**Comment by squarecandy** [comment]
sweet thanks!

**Comment by ntwb** [comment]
Apologies for his, just published v17.0.0 

> https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/releases/tag/17.0.0
